modules/ROOT/partials/configuration/text_patterns.adoc

```diff
@@ -74,11 +82,12 @@ Block patterns must have the following:
 
 * A `+start+`
 * A `+format+` or a `+cmd+`
+* A `+trigger+`
 ** If `+cmd+` is specified, an optional `+value+` property is allowed.
```

Comment:
Move this sub list item upper since it more relates to `command` configuration property